main.go

```diff
@@ -125,11 +126,35 @@ func decryptSymmetric(keyName string, ciphertext []byte) ([]byte, error) {
 	return resp.Plaintext, nil
 }
 
+func userInput() []byte {
+	fmt.Println("Enter the data you want to encrypt. END with CTRL+D")
+	rdr := bufio.NewReader(os.Stdin)
+	var lines []byte
+	for {
+		line, err := rdr.ReadBytes('\n')
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			log.Fatal("Error on input: %s", err)
+		}
+
+		// append scanned input to the array
+		lines = append(lines, line...)
+
+		if len(lines) == 0 {
```

Comment:
I was thinking about this, and I have a feeling that this case may never get hit. 

https://golang.org/pkg/bufio/#Reader.ReadBytes
> ReadBytes reads until the first occurrence of delim in the input, returning a slice containing the data up to **and including the delimiter**.

So if someone just enters a newline, it will append the newline byte into the lines slice. If we don't want newlines included, we should be filtering them out prior to appending to the slice

Reply:
Thats a good point, I went ahead and removed the default break on 0 line length, and added a bytes.trim() to clean up the user input.